bot/tasks/pennychat.py

```diff
@@ -240,6 +242,45 @@ def _penny_chat_details_blocks(penny_chat_invitation, mode=None):
         date_time_block
     ]
 
+    if penny_chat_invitation.video_conference_link:
```

Comment:
probably a nit pick, but rather than have 3 sections here, I'd just have a `{PREVIEW, INVITE, UPDATE}` section with a parenthetical `(A video link will be provided shortly before the chat starts)` and a `REMIND` section with the full details.

Reply:
So just remove the **Video Link** header from the invite and add parentheses?

Reply:
yup, just like you did - looks good